src/functions/blockchainFunctions.js

```diff
@@ -160,3 +160,30 @@ function historicalBalance_ (slug, from, to, address) {
     ]
   }))
 }
+
+function ethTopTransactions_ (slug, from, to, limit, transactionType) {
+  var results = getApiClient_().fetchEthTopTransactions(slug, from, to, limit, transactionType)
+  assertHasData_(results)
+
+  var headers = [
+    'Date',
+    'From Address',
```

Comment:
Shouldn't these names be concatenated? `FromAddress`, `FromAddressesIsExchange`, etc.?

Reply:
I don't think so at least. The other functions are made with spaces, I took them as a model to make these. If you say so I will make them camelCase